src/script/arch-update.sh

```diff
@@ -193,29 +193,29 @@ invalid_option() {
 icon_dir="/usr/share/icons/${name}"
 [ -d "/usr/local/share/icons/${name}" ] && icon_dir="/usr/local/share/icons/${name}"
 
-# Definition of the icon_checking function: Change icon to "checking"
-icon_checking() {
-	cp -f "${icon_dir}/${name}_checking.svg" "${icon_dir}/${name}.svg" || exit 3
+# Definition of the state_checking function: Change state to "checking"
+state_checking() {
+	echo "${name}_checking" > "${state_dir}/current_state"
 }
 
-# Definition of the icon_updates_available function: Change icon to "updates-available"
-icon_updates_available() {
-	cp -f "${icon_dir}/${name}_updates-available.svg" "${icon_dir}/${name}.svg" || exit 3
+# Definition of the state_update_available function: Change state to "updates-available"
+state_update_available() {
```

Comment:
```suggestion
state_updates_available() {
```

To match the name of the related state